src/services/refactors/extractLocal.ts

```diff
@@ -0,0 +1,1342 @@
+/// <reference path="../refactorProvider.ts" />

```

Comment:
Did you mean to include this file?

Reply:
I always forget that `checkout -f` doesn't delete new files. 😢